conda_libmamba_solver/plugin.py

```diff
@@ -5,26 +5,32 @@
 Entry points for the conda plugin system
 """
 
-from conda import plugins
+from conda.plugins import hookimpl
+
+try:
+    # FUTURE: Use this import only in conda 26.3+
+    from conda.plugins.types import CondaSolver, CondaSubcommand
+except ImportError:
+    from conda.plugins import CondaSolver, CondaSubcommand
```

Comment:
`CondaSolver`/`CondaSubcommand` has always been defined in `conda.plugin.types` so we don't need the fallback